tests/ui_tools/test_boxes.py

```diff
@@ -430,6 +430,8 @@ def test_keypress_typeahead_mode_autocomplete_key(self, mocker, write_box,
                                                       expect_footer_was_reset,
                                                       key):
         write_box.is_in_typeahead_mode = current_typeahead_mode
+        write_box.msg_write_box = mocker.Mock()
+        write_box.msg_write_box.edit_text = ''
```

Comment:
Is this related?

Reply:
Used in the older version I think - removed.